packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -245,34 +244,113 @@ QUnit.module('Chat', () => {
             });
         });
 
-        QUnit.test('Chat should pass messageTemplate to messageList on init', function(assert) {
-            const messageTemplate = () => { return $('<div>'); };
+        QUnit.test('messageTemplate should set bubble content on init', function(assert) {
```

Comment:
Strange for me to see this tests here while rendering is actually implemented on bubble level. But I don't mind, let it be a kind of integration tests.

But can we at least isolate all this test in a separate `messageTemplate` module?

Reply:
3c18eb83217a111cc7e4af6de5eef1636d87ee02